index.html

```diff
@@ -825,6 +825,28 @@ <h2>
           for a web page using CSS.
         </p>
       </section>
+      <section class='informative'>
+        <h2>
+          Interaction with Web Content Accessibility Guidelines
+        </h2>
+        <p>
+            The Web Content Accessibility Guidelines 2.1 specification [[WCAG21]] includes a Success Criterion (<a href="https://www.w3.org/TR/WCAG21/#orientation">SC 1.3.4</a>) related to screen orientation.
+        </p>
+
+        <p>
+            ​The intent of this Success Criterion is to ensure that all content and functionality is available regardless of the display
+            orientation (portrait or landscape). Some websites and applications automatically set the screen to a particular display
+            orientation and expect that users will respond by rotating their device to match.
+        </p>
+
+        <p>
+            However, some users may have their devices mounted in a fixed orientation (e.g. on the arm of a power wheelchair). Therefore,
+            websites and applications need to support both orientations by making sure content and functionality is available in
+            each orientation. While the order of content and method of functionality may have differences the content and functionality
+            must always be available. When a particular orientation is essential, the user needs to be advised of the orientation
```

Comment:
> While the order of content and method of functionality may have differences the content and functionality must always be available. 

I don't quite understand what this means. 

> When a particular orientation is essential, the user needs to be advised of the orientation ... 

It's a bit preachy... maybe:

When a particular orientation is essential, developers should inform users of the orientation requirement.

Reply:
>> While the order of content and method of functionality may have differences the content and functionality must always be available.
> I don't quite understand what this means.

It is a recognition that with CSS (i.e. flexbox, media queries) layout and content rendering may change when the orientation changes, however all content and functionality must be preserved. This may seem obvious on the surface, however... :) 

>> When a particular orientation is essential, the user needs to be advised of the orientation ...
> It's a bit preachy... maybe:
> When a particular orientation is essential, developers should inform users of the orientation requirement.

This is essentially an RFC 2119 MUST requirement. Replacing 'should' with 'must' would be preferred, however it is recognized that this is non-normative text in this Specification, so defer to Editors judgement.

Reply:
I think we usually try to avoid RFC 2119 key words in non-normative sections...